tests/integration/kubernetes/tests_common.sh

```diff
@@ -130,6 +130,39 @@ auto_generate_policy_enabled() {
 	[ "${AUTO_GENERATE_POLICY}" == "yes" ]
 }
 
+# adapt common policy settings for qemu-snp
+adapt_common_policy_settings_for_snp() {
```

Comment:
Is adapt_common_policy_settings_for_snp identical to adapt_common_policy_settings_for_sev?

If yes, let's keep just adapt_common_policy_settings_for_sev and call it on both KATA_HYPERVISOR branches.

Reply:
Updated. Thanks!